messagecard.go

```diff
@@ -248,8 +238,7 @@ func (mcs *MessageCardSection) AddImage(sectionImage ...MessageCardSectionImage)
 			return fmt.Errorf("cannot add empty image title")
 		}
 
-		mcs.Images = append(mcs.Images, &img)
-
+		mcs.Images = append(mcs.Images, &img) // nolint:scopelint
 	}
```

Comment:
Did I make a mistake here? I wasn't sure what the `scopelint` check did initially, but I looked it up (https://github.com/kyoh86/scopelint) and I see that it guards against unintentional variable reassignment/use. I'll take another look at this to make sure I've not accidentally done that here.

Reply:
I dug further into this and I think the linting error is probably valid. What do you think of this replacement code?

```diff
index 84c8c46..be90bc5 100644
--- a/messagecard.go
+++ b/messagecard.go
@@ -239,7 +239,7 @@ func (mcs *MessageCardSection) AddFactFromKeyValue(key string, values ...string)
 // provide a photo gallery inside a MessageCard section.
 func (mcs *MessageCardSection) AddImage(sectionImage ...MessageCardSectionImage) error {

-       for _, img := range sectionImage {
+       for index, img := range sectionImage {
                if img.Image == "" {
                        return fmt.Errorf("cannot add empty image URL")
                }
@@ -248,7 +248,7 @@ func (mcs *MessageCardSection) AddImage(sectionImage ...MessageCardSectionImage)
                        return fmt.Errorf("cannot add empty image title")
                }

-               mcs.Images = append(mcs.Images, &img)
+               mcs.Images = append(mcs.Images, &sectionImage[index])

        }

```

Instead of referencing the loop variable we're just using the index of the range loop to pull from the values passed into the method, which I presume are more stable based on the linter's complaint.